actions/delivery_runfolder_workflow.yaml

```diff
@@ -32,3 +32,8 @@ parameters:
     type: integer
     description: Configure the time to sleep in poll steps. Useful to reset when testing to make workflows run faster.
     default: 300
+  restrict_to_projects:
+    required: false
+    type: string
+    description: Name of specific project/projects to be delivered from runfolder. If several projects are specified separate project names on ,.
+    default: None
```

Comment:
I'd suggest changing this to `keep_all_projects`, or something along these lines. As `None` is a keyword in python, this might otherwise be confusing.